src/main/java/com/unosquare/adminCore/enums/EmployeeUserRoles.java

```diff
@@ -0,0 +1,28 @@
+package com.unosquare.adminCore.enums;
+
+import com.unosquare.adminCore.entity.EmployeeUserRole;
+
+public enum EmployeeUserRoles {
```

Comment:
Can we refactor to EmployeeRoles, with Employee representing the user in the system I think this will cause confusion.